docs/0install/setup.md

```diff
@@ -0,0 +1,28 @@
+The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
+
+## Prerequisites
+
+1. Install [0install](http://0install.net/injector.html) on your machine.
```

Comment:
> That means the JRE installation will be applying to all OS for now.

Meaning all users will have to install a JRE manually?  If so, it might be
better to make a separate feed for Macs.

On 22 June 2015 at 10:58, Alex Eng <notifications@github.com> wrote:

> In docs/0install/setup.md
> <https://github.com/zanata/zanata-client/pull/68#discussion_r32900504>:
>
> > @@ -0,0 +1,28 @@
> > +The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
> > +
> > +## Prerequisites
> > +
> > +1. Install [0install](http://0install.net/injector.html) on your machine.
>
> Cofirmed that with the jre dependency in place, Mac won't be able to run
> even if it has local valid JRE installed. So for now, I've commented out
> the dependency until they fix it in the 0install repo. That means the JRE
> installation will be applying to all OS for now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-client/pull/68/files#r32900504>.
>